cmd/kubectl-gadget/undeploy.go

```diff
@@ -234,7 +234,10 @@ again:
 
 		ctx, cancel := context.WithTimeout(context.Background(), time.Duration(timeout)*time.Second)
 		defer cancel()
-		watchtools.Until(ctx, list.ResourceVersion, watcher, conditionFunc)
+		_, err := watchtools.Until(ctx, list.ResourceVersion, watcher, conditionFunc)
+		if err != nil {
+			errs = append(errs, fmt.Sprintf("Failed waiting for IG to be removed: %s", err))
```

Comment:
I would instead use `namespace` or `Inspektor Gadget`:

```suggestion
			errs = append(errs, fmt.Sprintf("Failed waiting for namespace to be removed: %s", err))
```

And then error should be printed:

```go
return fmt.Errorf("error removing Inspektor Gadget:\n%s", strings.Join(errs, "\n"))
```

Reply:
Fixed. Thanks for the suggestions.

Reply:
Fixed. Thanks for the suggestions.